eos-chrome-plugin-update-upstream

```diff
@@ -21,6 +21,21 @@ DEST_DIR=/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater
 WGET_COMMAND="wget -c --tries=5"
 DAYS_CHECK=7
 
+# Data needed for the Flash plugin, from Adobe
+FLASH_FALLBACK_VERSION="23.0.0.185"
+FLASH_FALLBACK_URL="https://fpdownload.adobe.com/pub/flashplayer/pdc/${FLASH_FALLBACK_VERSION}/flash_player_ppapi_linux.$(arch).tar.gz"
+FLASH_FALLBACK_SHA256SUM="3f7a13a615d3cf19e5feb02485e12e3785a6ef07cd7ae32072cd860468fd322b"
+
+FLASH_LKGV_FILE_URL="https://raw.githubusercontent.com/endlessm/eos-chrome-plugin-updater/master/flashplugin-$(arch).lkgv"
```

Comment:
How secure is it to reach back to github to get the URL and SHA256SUM?  Wouldn't that be open to a simple attack of injecting an arbitrary URL with the associated SHA256SUM?  If not really secure, why not simply download the latest from Adobe rather than specify a version?  Or, is there no way to do this without calling out a specific version?